types/react/index.d.ts

```diff
@@ -1090,8 +1090,7 @@ declare namespace React {
      * @version 16.8.0
      * @see https://reactjs.org/docs/hooks-reference.html#usecallback
      */
-    // TODO (TypeScript 3.0): <T extends (...args: never[]) => unknown>
```

Comment:
Not sure if I should keep that comment?
If the `Function` approach works, why would we change later to the `never[]`?

Reply:
Makes sense to remove.